modules/web/js/debugger/debug-manager.js

```diff
@@ -137,13 +137,26 @@ define(['require', 'jquery', 'backbone', 'lodash', 'event_channel', './channel',
         return this.enable;
     };
 
-    DebugManager.prototype.getDebugPoints = function () {
-        return this.debugPoints;
+    DebugManager.prototype.getDebugPoints = function (fileName) {
```

Comment:
added doc comments in this PR https://github.com/ballerinalang/composer/pull/1996